.github/workflows/static_analysis.yml

```
@@ -0,0 +1,18 @@
+name: Static Analysis
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v1
```

Seems there is already a v2
Did you test it before?
```suggestion
        uses: actions/setup-python@v2
```

---

I wasn't aware there was a new one